configure_project/variables.tf

```diff
@@ -0,0 +1,39 @@
+variable "project_name" {
+  description = "Name of the Watson project to create"
```

Comment:
I'd suggest to put more detailed description of the variable overall (giving some more hint on how the object that are refered to are used) - not necessarily in this PR but in the future. @SirSpidey may be able to help review the wording, etc

Reply:
sure, @SirSpidey please help